exercises/circular-buffer/description.md

```diff
@@ -4,39 +4,55 @@ A circular buffer, cyclic buffer or ring buffer is a data structure that uses a
 
 A circular buffer first starts empty and of some predefined length.
 For example, this is a 7-element buffer:
-<!-- prettier-ignore -->
```

Comment:
Removing the comment and reformatting didn't produce any changes. But just to be safe, I'll wrap those in backticks. A 4 space intent is equivalent to backticks, but it's not as clear in its intention.